_posts/2022-03-24-vegastrike-0.8.0-release.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Vega Strike 0.8.0 Released!"
+description: "New Releases: Vega Strike 0.8.0, VS:UtCS 0.8.0"
+author: BenjamenMeyer
+date: 2022-03-24 00:00:00 +0000
+categories: website announcements releases community
+excerpt_separator: <!--more-->
+---
+It is with great pleasure that the Vega Strike Development Team announces the release of Vega Strike 0.8.0 and VS:Upon The Coldest Sea 0.8.0.
+<!--more-->
+
+Join our protagonist Deucalion in adventures around the universe as a bounty hunter, mercenary, or merchant flying through the
+stars. Meet the deadly Aera and mysterious Rlaan. Battle with the Confederation. Run alongside Pirates. Or just buy and sell as a merchant while you
+explore. The universe and adventure are yours.
+
+## What's changed?
+
+Since our last release in April 2021 of Vega Strike 0.7.0 we've continued to refactor the code and make general improvements.
+
+We have also made the Python3 Support the official build. We are no longer providing Python2 builds; if you need Python2 it is still
+available but you must make your own build from source. Python2 support will be completely removed in our next release (0.9.0).
+
+Some changes to OpenGL have also required that we provide two separate builds on Linux Platforms. Unfortunately this requires making
+separate builds due to linking against libraries that we can't do otherwise. Our old methods of using OpenGL are available using the
+"LEGACY" builds; while the newer support is provided using the "GLVND" builds.
+
+The Vega Strike Engine now supports a Versioning API that provides the release version (e.g 0.8.0) and the Engine API Version (e.g 1).
+The Engine API version works much like the Android API Version - it's a simple incrementing unsigned integer value. If the API does
+not exist (import error in Python) then the Engine API Version is zero (0) and the release version is at most 0.7.0, but can also
+be any earlier version. We highly recommend that game assets developers gate on this API as going forward we are going to start
+introducing changes where games will need to rely on the Engine API Version value to determine that they are compatible with the
+engine. Aside from the introduction of this API there is no difference in the Python API provided from 0.7.0.
+
+The Vega Strike Developers are also putting in place a standard binary naming convension of `vegastrike-<tool` or `libvegastrike-<library>`.
```

Comment:
`convention` not `convension`. Also, the closing angle bracket is missing after `tool`.